src/new-client/src/app/site/[site]/details/site-details-page.tsx

```diff
@@ -16,7 +16,13 @@ const SiteDetailsPage = async ({ site, permissions }: Props) => {
   const accessibilityAttributeDefinitions = attributeDefinitions.filter(ad =>
     ad.id.startsWith('accessibility'),
   );
-  const siteAttributeValues = await fetchSiteAttributeValues(site.id);
+  const siteDetails = await fetchSite(site.id);
+  const informationForCitizenAttribute =
+    siteDetails === undefined
+      ? ''
```

Comment:
As with siteMoniker, you should now be able to ditch the undefined check here I think?